radicle/src/cob/issue.rs

```diff
@@ -59,6 +59,21 @@ pub enum State {
     Open,
 }
 
+impl From<String> for State {
+    fn from(s: String) -> Self {
+        match s.as_str() {
+            "closed_other" => Self::Closed {
+                reason: CloseReason::Other,
+            },
+            "closed_solved" => Self::Closed {
+                reason: CloseReason::Solved,
+            },
+            "open" => Self::Open,
+            _ => Self::default(),
+        }
+    }
+}
```

Comment:
I would say the same here, we should use `FromStr`.

It irks me that the `Display` and `FromStr` will differ though, so we can't say `state.to_string().parse() == state`. Not sure what a good solution would be though.